### PR TITLE
chore: use slot IDs for banner ads instead of aspect ratios

### PR DESCRIPTION
**Pull request by roalonso**
### Changes

Document new `slotId` as implemented by https://github.com/Topsort/auction-engine/pull/1555.

### CODE REVIEW

**Comment by PabloReszczynski** [comment]
Same comment I made to @rmellat 's PR: We should support both `slotId` xor `aspectRatio` while we transition between APIs. Otherwise, we would be forced to update our api version to the next major number.

File: topsort-api-v2.yml [hunk 1]
```diff
@@ -319,20 +319,21 @@ components:
...
           $ref: '#/components/schemas/Device'
-        aspectRatio:
+        slotId:
           type: string
-          pattern: '^\d+:\d+$'
           description: >
-            The aspect ratio for a specific slot for a banner placement in the marketplace app.
-            The aspect ratio for the category (or home page, if no category ID was provided in the request) must have been previously set up in the Topsort app.
+            The ID of the region for which this auction will be run for.
+            It must match the values provided when creating the banner campaign.
```
Comment (sk-) [inline topsort-api-v2.yml:326]: Not sure if we should have a comment like this. Also I'm not even sure this valu is safe to be an id, as according to https://stackoverflow.com/questions/1077084/what-characters-are-allowed-in-dom-ids it must start with a letter.

File: topsort-api-v2.yml [hunk 2]
```diff
@@ -117,7 +117,7 @@ paths:
                       location: New York
                   - type: banners
                     slots: 1
-                    aspectRatio: '4:1'
+                    slotId: 254b6dfc-3447-4ddf-a270-ed23101023ef
```
Comment (sk-) [inline topsort-api-v2.yml:120]: Are slot-ids going to be uuids? That could be error prone for developers.

**Review by sk- — CHANGES_REQUESTED** [review]

File: topsort-api-v2.yml (hunk 2 above)
Comment (PabloReszczynski) [reply topsort-api-v2.yml:120]: Not necessarily, it will be provided by the marketplace, so it could be a UUID or a number or anything really

**Review by PabloReszczynski — APPROVED** [review]

File: topsort-api-v2.yml (hunk 2 above)
Comment (roalonso) [reply topsort-api-v2.yml]: Changed to a kebab case string.

File: topsort-api-v2.yml (hunk 1 above)
Comment (roalonso) [reply topsort-api-v2.yml]: I agree with both, I ended up using banner placement.

**Pull request merged**